QuizWhois/Domain/Entity/Question.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Domain.Entity
+{
+    public class Question : Base.Base
+    {
+        public string QuestionText { get; set; }
+        public string Answer { get; set; }
```

Comment:
Maybe we should call it "CorrectAnswer" or discuss making it an array ie question could have more than one correct answer

Reply:
Let's rename for now